python/mxnet/recordio.py

```diff
@@ -274,16 +274,17 @@ def write_idx(self, idx, buf):
 IRHeader = namedtuple('HEADER', ['flag', 'label', 'id', 'id2'])
 """An alias for HEADER. Used to store metadata (e.g. labels) accompanying a record.
 See mxnet.recordio.pack and mxnet.recordio.pack_img for example uses.
+
 Parameters
 ----------
-flag : int
-    Available for convenience, can be set arbitrarily.
-label : float or an array of float
-    Typically used to store label(s) for a record.
-id: int
-    Usually a unique id representing record.
-id2: int
-    Higher order bits of the unique id, should be set to 0 (in most cases).
+    flag : int
```

Comment:
formatting for IRHeader is lost, see http://mxnet.io/api/python/io.html#mxnet.recordio.IRHeader.
How do we fix the formatting ?